thirdeye/thirdeye-frontend/app/helpers/color-delta.js

```diff
@@ -9,7 +9,7 @@ import Ember from 'ember';
 export function colorDelta([value = 'N/A']) {
   let sign = true;
   if (value ===  'N/A') { return; }
```

Comment:
If you're gonna put it on one line anyway, i think `if (value === 'N/A') return` works and looks cleaner.